src/content/docs/en/recipes/bun.mdx

```diff
@@ -2,86 +2,17 @@
 title: Use Bun with Astro
 description: Learn how to use Bun with your Astro site.
 type: recipe
-i18nReady: true
+i18nReady: false
 ---
 
-[Bun](https://bun.sh/) is a runtime for JavaScript built for speed. See [Bun's documentation](https://bun.sh/docs) for more information.
+[Bun](https://bun.sh/) is a runtime for JavaScript built for speed.
 
-Using Bun with Astro still is experimental. Some integrations may not work as expected. If you have [any problems using Bun](https://github.com/oven-sh/bun/issues/new/choose), please [open an issue on GitHub](https://github.com/withastro/astro/issues/new/choose)!
+Using Bun with Astro is now officially supported but is still very new. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. 
```

Comment:
I think this is liable to be misconstrued. Officially supported by whom? It's not us.
```suggestion
Bun recently made their first stable release. However, using Bun with Astro is still very new. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. 
```

Reply:
+1 although maybe `edge cases` or `incompatibilities` is more granular than `rough edges`?

Reply:
They are still getting bug reports for fatal crashes, so I would prefer being general.

https://github.com/oven-sh/bun/issues?q=is%3Aissue+is%3Aopen+core+dumped